lib/saml/user_attributes/ssoe.rb

```diff
@@ -91,7 +92,11 @@ def mhv_icn
       end
 
       def mhv_correlation_id
-        safe_attr('va_eauth_mhvien')
+        safe_attr('va_eauth_mhvuuid') || safe_attr('va_eauth_mhvien')
```

Comment:
Just double checking that it's `va_eauth_mhvien` IEN and not ICN

Reply:
Yep, mhvien, yet another alias for mhv account id aka mhv uuid aka mhv correlation id.